### PR TITLE
[QG][KIM] Application that generates loads by creating or deleting Runtime CR

### PR DESCRIPTION
**Pull request by mvshao**
**Description**
The  program is used to generate performance test loads by creating or deleting Runtime CR in a given Kubernetes cluster. 

Changes proposed in this pull request:

- created an application that generates test Runtime CR in amount provided by the user
- supports local testing and CI/CD environment

**Related issue(s)**
#14

### CODE REVIEW

**Review by koala7659 — COMMENTED** [review]
Please check my comments

File: hack/performance/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Runtime Load Generator
+
+## Overview
+
+The `rt-load` program is used to generate performance test loads by creating or deleting Runtime resources in a Kubernetes cluster. All of the Runtime Custom Resources created by this program will be linked to the same Gardener cluster.
```
Comment (grego952) [inline hack/performance/README.md:5]:
```suggestion
The `rt-load` program generates performance test loads by creating or deleting runtime resources in a Kubernetes cluster. All of the runtime custom resources created by this program are linked to the same Gardener cluster.
```

File: hack/performance/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+```sh
+rt-load <command> [options]
+```
+
+## Commands
+
+### `create`
+
+Creates a specified number of Runtime resources.
```
Comment (grego952) [inline hack/performance/README.md:28]:
```suggestion
Creates a specified number of runtime resources.
```

File: hack/performance/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+#### Example
+
+```sh
+./rt-load create --load-id my-load --name-prefix my-runtime --kubeconfig /path/to/kubeconfig --rt-number 10 --rt-template /path/to/template.yaml
+```
+
+### `delete`
+
+Deletes Runtime resources based on the specified load ID.
```
Comment (grego952) [inline hack/performance/README.md:47]:
```suggestion
Deletes runtime resources based on the specified load ID.
```

**Review by grego952 — APPROVED** [review]

File: hack/performance/cmd/cmd.go [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+		worker, err := action.NewWorker(*loadID, *namePrefix, *kubeconfig, *rtNumber, parsedRuntime)
+		return Create, worker, err
+	case "delete":
+		deleteCmd.Parse(os.Args[2:])
+		if *loadIDDelete == "" || *kubeconfigDelete == "" {
+			fmt.Println("all flags --load-id and --kubeconfig are required")
+			deleteCmd.Usage()
+			os.Exit(1)
+		}
+		fmt.Printf("Deleting load with ID: %s, Kubeconfig: %s\n", *loadIDDelete, *kubeconfig)
```
Comment (koala7659) [inline hack/performance/cmd/cmd.go:87]:
```suggestion
		fmt.Printf("Deleting load with ID: %s, Kubeconfig: %s\n", *loadIDDelete, *kubeconfigDelete)
```

**Review by koala7659 — COMMENTED** [review]
Almost ready just one small suggestion

File: hack/performance/cmd/cmd.go (hunk 4 above)
Comment (mvshao) [reply hack/performance/cmd/cmd.go:87]: Thx, for catching this

**Review by koala7659 — APPROVED** [review]
LGTM

**Review by grego952 — APPROVED** [review]

**Pull request merged**